### DISCUSSION TITLE
Inquiry abaout security

### DISCUSSION THREAD

**Discussion by Pathfinder0920** [jgraph/drawio-desktop#1869] [Q&A]
Hi,

We are considering introducing a desktop version of your draw.io.
Our security department wants to check the security.
Do you have the security confirmation result of the safety or the security certificate of the third party?

Attached is the page requested by our security department for reference.
https://ja.confluence.atlassian.com/sourcetreekb/security-advisories-900820352.html

Sincerely

**Comment by davidjgraph** [comment]
That's a link to sourcetree documentation, what does that have to do with this project?
